public/index.html

```diff
@@ -79,6 +79,22 @@
     />
 
     <link rel="stylesheet" href="fonts.css" type="text/css" />
+    <script>
+      // Redirect Excalidraw plus users with set auto-redirect to app.excalidraw.com
+      // only root path - any other shared, json... paths are not redirected
+      //
+      // THIS SHOULD BE IDEALY DONE ON THE SERVER.
+      // However current Excalidraw is served from Vercel without any backend so trying to redirect as soon
+      // we can.
```

Comment:
we can redirect based on a cookie in Vercel, but the problem is we need to check `location.hash` which you obviously can't do on any server